README.md

```diff
@@ -0,0 +1,50 @@
+# NevolutionX
+Original Xbox dashboard created with nxdk.
+
+![Screenshot](https://en.wikipedia.org/wiki/Hammer_and_sickle#/media/File:Hammer_and_sickle_red_on_transparent.svg "Screenshot")
+
+## Motivation
+As the XboxDev community grows, the need for an open-source, nxdk based dashboard arises. There are other dashboard alternatives i.e. XBMC but none of them are build using nxdk.
+
+## Features
+- [x] Basic Menu
+- [x] Build-in FTP server
+	- [x] IPv4 & IPv6
+	- [ ] TLS
+- [x] Application launcher
+- [x] DVD launcher
+- [ ] Themes
+
+## Build
+
+In order to build NevolutionX you'll need to install [nxdk](https://github.com/XboxDev/nxdk) first.
+To build NevolutionX you simply clone the repository and run `cmake` followed by `make`. You may need to adjust the installation location of nxdk in the `Makefile` to your needs. The default is `NXDK_DIR = $(CURDIR)/../nxdk` which expects nxdk right next to NevolutionX.
```

Comment:
`cmake` is not the way to go for the xbox build (yet). We're probably better off describing the `make` route.
The part about NevolutionX and nxdk sharing the parent directory is probably good to keep. :+1:

Reply:
Whats the `make` route? 😅

Reply:
Basically just "Clone NevolutionX into the same parent folder as nxdk, run `make` inside the NevolutionX root dir`. :)